Makefile

```diff
@@ -51,3 +54,28 @@ check-style:
 verify-gomod:
 	$(GO) mod download
 	$(GO) mod verify
+
+# Draft a release
+release:
+	@if [[ -z "${NEXT_VER}" ]]; then \
+		echo "Error: NEXT_VER must be defined"; \
```

Comment:
Perhaps we can add something like - "Run make release NEXT_VER=v1.0.1" to make it clearer